### PR TITLE
1.3 RC Changelog updates

### PR DESCRIPTION
**Pull request by MadelineMurray**
1.3 RC Changelog updates

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +10,32 @@ From v1.2, Besu requires Java 11.  Besu on Java 8 is no longer supported.
...
+- Allow private contracts to access public state. [\#9](https://github.com/hyperledger/besu/pull/9) (thanks to [josh-richardson](https://github.com/josh-richardson))
+- Documentation updates include: 
+  - Added [`retesteth`](https://besu.hyperledger.org/en/latest/Reference/CLI/CLI-Subcommands/#retesteth) subcommand 
+  - Added [`debug_accountRange`](https://besu.hyperledger.org/en/latest/Reference/API-Methods/#debug_accountrange) JSON-RPC API method 
+  - Clarified purpose of [static nodes](https://besu.hyperledger.org/en/latest/HowTo/Find-and-Connect/Managing-Peers/#static-nodes) 
+
+### Technical Improvements 
+
+- Return enclave key instead of private transaction hash [\#53](https://github.com/hyperledger/besu/pull/53) (thanks to [iikirilov](https://github.com/iikirilov))
+- Fix some mark sweep pruner bugs where nodes that should be kept were being swept  [\#50](https://github.com/hyperledger/besu/pull/50)
```
Comment (NicolasMassart) [inline CHANGELOG.md]: I would not use the word "some" here as it makes the sentence a bit vague.

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,6 +10,32 @@ From v1.2, Besu requires Java 11.  Besu on Java 8 is no longer supported.
...
+
+### Technical Improvements 
+
+- Return enclave key instead of private transaction hash [\#53](https://github.com/hyperledger/besu/pull/53) (thanks to [iikirilov](https://github.com/iikirilov))
+- Fix some mark sweep pruner bugs where nodes that should be kept were being swept  [\#50](https://github.com/hyperledger/besu/pull/50) 
+- Clean up BesuConfiguration construction [\#51](https://github.com/hyperledger/besu/pull/51) 
+- Private tx nonce errors return same msg as any tx [\#48](https://github.com/hyperledger/besu/pull/48) 
+- Fix default logging [\#47](https://github.com/hyperledger/besu/pull/47) 
+- Introduce virtual operation. [\#45](https://github.com/hyperledger/besu/pull/45) 
+- Downgrade some RocksDBPlugin Logging Levels [\#44](https://github.com/hyperledger/besu/pull/44)
```
Comment (NicolasMassart) [inline CHANGELOG.md]: I would not use the word "some" here as it makes the sentence a bit vague.

File: CHANGELOG.md (hunk 1 above)
Comment (MadelineMurray) [reply CHANGELOG.md]: Fixed

File: CHANGELOG.md (hunk 2 above)
Comment (MadelineMurray) [reply CHANGELOG.md]: Fixed

File: CHANGELOG.md [hunk 3]
```diff
@@ -9,6 +10,36 @@ From v1.2, Besu requires Java 11.  Besu on Java 8 is no longer supported.
...
+### 1.3 RC 
+
+### Breaking Change 
+
+- Disallow comments in Genesis JSON file. [\#49](https://github.com/hyperledger/besu/pull/49) 
+
+### Additions and Improvements
+
+- Store db metadata file in the root data directory. [\#46](https://github.com/hyperledger/besu/pull/46) 
+- Add `--targat-gas-limit` command line option. [\#24](https://github.com/hyperledger/besu/pull/24)(thanks to new contributor [cfelde](https://github.com/cfelde))
```
Comment (aldoborrero) [inline CHANGELOG.md]: @MadelineMurray there's a typo on `--target-gas-limit` (`--targat-`)

Comment (EdJoJob) [inline CHANGELOG.md]:
```suggestion
- Add `--target-gas-limit` command line option. [\#24](https://github.com/hyperledger/besu/pull/24)(thanks to new contributor [cfelde](https://github.com/cfelde))
```

**Pull request merged**